### PR TITLE
New estimated memory and disk plan properties.

### PR DESCRIPTION
**Pull request by stbof**
https://cockroachlabs.atlassian.net/browse/DOC-1679

Other misc fixes.

### CODE REVIEW

File: v22.1/explain.md [hunk 1]
```diff
@@ -44,22 +44,22 @@ The user requires the appropriate [privileges](security-reference/authorization.
...
 
 Detail | Description
 -----------|-------------
 Global properties | Properties that apply to the entire query plan. Global properties include `distribution` and `vectorized`.
-Statement plan tree | A tree representation of the hierarchy of the statement plan.
-Node details | The properties, columns, and ordering details for the current statement plan node in the tree.
-Number of rows | The number of rows affected by the query.
+Statement plan tree properties | A tree representation of the hierarchy of the statement plan.
+index recommendations: N | Number of index recommendations followed by a list of index actions and SQL statements to perform the actions.
+(N rows) | The number of rows affected by the query.
```
Comment (mgartner) [inline v22.1/explain.md:62]: I think we could remove this. See my other comment in explain-analyze.md.

**Review by mgartner — APPROVED** [review]
LGTM.

@yuzefovich can you quickly review the changes in explain-analyze.md related to memory and disk usage?

**Review by yuzefovich — COMMENTED** [review]
:lgtm:

> Reviewed 2 of 2 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/13999#-:-N42pbqC-uhXBNILtjvt:bnfp4nl)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @stbof)

File: v22.1/explain.md [hunk 2]
```diff
@@ -133,7 +132,8 @@ The output also describes a set of properties, some global to the query, some sp
 
 - `index recommendations: 1`
 
-    A recommendation to create an index to eliminate the full scan of the table. The recommendation is to create an index on the `rides` table and store the columns `vehicle_city`, `rider_id`, `vehicle_id`, `start_address`, `end_address`, `start_time`, `end_time`.
+    The number of index recommendations, followed by the recommendation and statement. The recommendation to create an index on the `rides` table and store the columns `vehicle_city`, `rider_id`, `vehicle_id`, `start_address`, `end_address`, `start_time`, `end_time`, will eliminate the full scan of the `rides` table.
```
Comment (nickvigilante) [inline v22.1/explain.md:135]:
```suggestion
    The number of index recommendations, followed by the recommendation and statement. The recommendation to create an index on the `rides` table and [store](indexes.html#storing-columns) the `vehicle_city`, `rider_id`, `vehicle_id`, `start_address`, `end_address`, `start_time`, and `end_time` columns will eliminate the full scan of the `rides` table.
```

**Comment by nickvigilante** [comment]
@stbof LGTM!

**Pull request merged**